docs/diagrams/src/edpm.puml

```diff
@@ -28,20 +28,20 @@ folder AEEconfigmap #line.bold
 
 user --> OCPCR #line:red;line.bold;text:red  : applies
 user --> ODPCR #line:red;line.bold;text:red  : applies
-OCPCR --> OSOp #line:brown;line.bold
-ODPCR --> OSOp #line:brown;line.bold
+OCPCR --> OSOp #line:brown;line.bold;text:brown : consumed
+ODPCR --> OSOp #line:brown;line.bold;text:brown : consumed
 OSOp --> NCR #line:purple;line.bold;text:purple  : creates
 OSOp --> ODPNCR #line:purple;line.bold;text:purple  : creates
 OSOp --> ODPRCR #line:purple;line.bold;text:purple  : creates
-ODPNCR --> DOp #line:brown;line.bold
-ODPRCR --> DOp #line:brown;line.bold
-NCR --> NOp #line:brown;line.bold
+ODPNCR --> DOp #line:brown;line.bold;text:brown : consumed
+ODPRCR --> DOp #line:brown;line.bold;text:brown : consumed
+NCR --> NOp #line:brown;line.bold;text:brown : consumed
 DOp --> AEECR #line:purple;line.bold;text:purple  : creates
 DOp --> NECCR #line:purple;line.bold;text:purple  : creates
-AEECR --> AEEOp #line:brown;line.bold
-NECCR --> NOp #line:brown;line.bold
-AEEOp --> AJI #line:blue;line.bold
-AEEOp --> AJNC #line:blue;line.bold
+AEECR --> AEEOp #line:brown;line.bold;text:brown : consumed
+NECCR --> NOp #line:brown;line.bold;text:brown : consumed
```

Comment:
i woudl use manages in instead of consumes and i think the arrow is reversed.

so

NOp --> NECCR #line:brown;line.bold;text:brown : manages

and i think the same applies to all the lines you added consumed too

i think the relation ship should be reversed and we should use manages instead of consumed.

Reply:
actully reconsiles might also be a better name then manages